docroot/modules/custom/dcc_gtd_scheduler/src/Form/SchedulerForm.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace Drupal\dcc_gtd_scheduler\Form;
+
+use Drupal\Core\Datetime\Element\Datetime;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+
+/**
+ * Create a form similar to 'Drupal Training Scheduler' one, which allows
```

Comment:
Class, methods, function comments should on a singe line. If more details are required, you should leave after the first comment an empty line and add the details. Also, a comment line should not exceed 80 chars.